src/main/fc/fc_msp_box.c

```diff
@@ -54,7 +54,7 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXGPSHOME, "GPS HOME", 10 },
     { BOXGPSHOLD, "GPS HOLD", 11 },
     { BOXPASSTHRU, "PASSTHRU", 12 },
-    { BOXBEEPERON, "BEEPER", 13 },
+    { BOXBEEPERON, "PIEZO BEEPER", 13 },
```

Comment:
Can you rename this back to beeper? Stipulating that users have to use a piezo beeper for this to work is wrong.

Reply:
Sounds good, just was trying to differentiate then.

Reply:
What else do people use besides piezos?

Reply:
Any kind of buzzers really  -  there's even boards that work with a normal speaker and use PWM to modulate.